pwiz_tools/Skyline/EditUI/AssociateProteinsDlg.Designer.cs

```diff
@@ -80,7 +80,6 @@ private void InitializeComponent()
             // btnCancel
             // 
             resources.ApplyResources(this.btnCancel, "btnCancel");
-            this.btnCancel.DialogResult = System.Windows.Forms.DialogResult.Cancel;

```

Comment:
We usually leave the DialogResult for the cancel button. If you remove it, I think you need a click action handler. What happens when you click the Cancel button?

Reply:
With the CancelButton property set but no DialogResult set on the button, clicking the button does close the dialog and return DialogResult.Cancel. But TBH I didn't even think about it, I just removed the DialogResult from both buttons thinking they'd both be redundant. But I can see why you wouldn't expect so since DialogResult.Cancel is never returned explicitly. And when I tested removing the DialogResult = DialogResult.OK from the OkDialog() function, the OK button no longer closes the form. So maybe those Accept/CancelButton properties only affect the DialogResult if there's no other click handler.

Reply:
Maybe CancelButton is special in that way. I had always thought they really only determined what button got a clicked message when you press Enter = Accept, Esc = Cancel. Clearly, I had never worked out whether those settings might also set DialogResult in the absence of an OnClicked handler.